src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const allCountriesPopulation = [...document.querySelectorAll('.population')];
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const totalPopulationNumber = allCountriesPopulation
+  .map(country => Number(country.textContent.split(',').join('')))
+  .reduce((prev, acc) => prev + acc, 0);
+const averagePopulationNumber = totalPopulationNumber
+  / allCountriesPopulation.length;
```

Comment:
let's clean up your code a bit
```suggestion
const totalPopulationNumber = allCountriesPopulation
  .map(country => +country.textContent.split(',').join(''))
  .reduce((prev, acc) => prev + acc, 0);
  
const averagePopulationNumber = totalPopulationNumber
  / allCountriesPopulation.length;
```